Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -33,6 +34,8 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
 
   const activeBot = skillId ?? projectId;
 
+  useSurveyNotification();
```

Comment:
If we ever decide the notification should be triggered elsewhere, this hook is all that will need to be moved there.